LanguageFeatures/Augmentation-libraries/augmenting_types_A10_t02_lib.dart

```diff
@@ -49,3 +49,23 @@ augment enum E {
 // [analyzer] unspecified
 // [cfe] unspecified
 }
+
+augment extension ExtA {
+  int get foo => 42;
+//        ^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+  void set bar(String _) {}
+//         ^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  int operator -(int other) => other;
```

Comment:
Same thing here: No compile-time errors here, but this operator `-` is not so easy to call outside `ExtA`.